jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -91,6 +91,7 @@ protected void startNotificationThread() {
 		}
 	}
 
+/*[IF JAVA_SPEC_VERSION < 19]*/
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
```

Comment:
This should be updated to advise use of something that isn't going away in Java 20.